### PR TITLE
Add link to interview/feedback sessions resource

### PR DESCRIPTION
**Pull request by reese-lee**
Adds a link to the newly added page about user interview/feedback sessions (https://github.com/open-telemetry/opentelemetry.io/pull/2297).

### CODE REVIEW

**Review by chalin — APPROVED** [review]
LGTM once #2297 is merged :)

**Comment by svrnm** [comment]
@reese-lee you could combine the 2 PRs into one

**Comment by svrnm** [comment]
I fixed the conflict, @chalin I think this is not blocked anymore?

**Comment by chalin** [comment]
> I fixed the conflict, @chalin I think this is not blocked anymore?

I'm still seeing a conflict -- I can't even rebase from the web i/f. Lemme try to fix that ...

**Comment by chalin** [comment]
Thanks for following up on this @svrnm.

**Pull request merged**
